.github/workflows/cd.yml

```
@@ -0,0 +1,38 @@
+name: CD
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Hmmm, this did not run on the PR, and did not give me the button to approve running it (since you are making the PR from a fork), either… do these maybe need to be:

```suggestion
  workflow_dispatch: {}
  pull_request: {}
```